src/DetailsView/components/export-dialog.tsx

```diff
@@ -48,6 +56,8 @@ export const ExportDialog = NamedFC<ExportDialogProps>('ExportDialog', props =>
     };
 
     const fileURL = props.deps.fileURLProvider.provideURL([props.html], 'text/html');
+    const exportService = ReportExportServiceProviderImpl.forKey(format);
```

Comment:
`ReportExportServicePRoviderImpl` is being use as a global dependency. Among other unwanted side effects, this forces the unit test to not be properly isolated (by depending on a concrete class/instance/implementation that's not the unit being tested).
One way to solve this is to follow the "`deps` pattern" we have across the repo:
Add a new property under `ExportDialogDeps` where we can pass an instance of the `ReportExportServiceProvider`. For the extension itself, this instance will be `ReportExportServiceProviderImpl` but for the unit test it can be a mock which does not need to change every time `ReportExportServicePRoviderImpl` changes, making the unit test properly isolated of other implementation details.